DocumentDataAPI/DocumentDataAPI/Controllers/SearchController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using DocumentDataAPI.Data.Repositories;
+using DocumentDataAPI.Models;
+
+using Microsoft.AspNetCore.Mvc;
+using System.Net.Mime;
+using DocumentDataAPI.Data.Services;
+
+namespace DocumentDataAPI.Controllers;
+
+[ApiController]
+[Route(RoutePrefixHelper.Prefix + "/search")]
+[Produces(MediaTypeNames.Application.Json)]
+public class SearchController : ControllerBase
+{
+    private readonly ILogger<SearchController> _logger;
+    private readonly ISearchRepository _repository;
+    private readonly ILemmatizerService _lemmatizerService;
+
+    public SearchController(ILogger<SearchController> logger, ISearchRepository repository, ILemmatizerService lemmatizerService)
+    {
+        _logger = logger;
+        _repository = repository;
+        _lemmatizerService = lemmatizerService;
+    }
+
+    /// <summary>
+    /// Retrieves a list of all documents relevant to a given search (a list of comma-separated words and delimiting parameters for a document).
```

Comment:
Er tanken her så at en søgning skal have følgende format: "Aalborg, Fodbold, Kamp, Uafgjort"?

Reply:
Altså uden mellemrum ja - det er bare en kommasepareret liste af ord. Så dit eksempel vil bare være "aalborg,fodbold,kamp,uafgjort"

Reply:
Og 'case' er sådan set ligegyldigt, for det bliver konverteret til lower-case af lemmatizeren alligevel